condensation/utils.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+from cfglib import CONF
+import yaml
+
+
+def read_file(path):
+    """This function reads yaml / json files"""
+
+    # check if path has extension
+    if '.' not in path:
+        raise RuntimeError("File {path} has no extension".format(path=path))
+    extension = path.split(".")[-1]
+
+    # check if extension is json or yml
+    extension_map = {"json": json, "yml": yaml}
+    if extension not in extension_map:
+        raise RuntimeError(
+            "File extension of {path} is not yaml/json".format(path=path))
+
+    # check if path exists
+    if not os.path.exists(path):
+        raise RuntimeError("File {path} not found".format(path=os.path.abspath(path)))
```

Comment:
This check seems redundant, you'll get `IOError` on {json,yaml}.load(descriptor) with more appropriate contents.